actions/ostree_commit_action.go

```diff
@@ -22,10 +29,18 @@ type (https://ostree.readthedocs.io/en/latest/manual/repo/#repository-types-and-
 Optional properties:
 
 - subject -- one line message with commit description.
+
+- collection-id -- Collection ID ref binding (require libostree 2018.6).
+
+- ref-binding -- enforce that the commit was retrieved from one of the branch names in this array.
+  If 'collection-id' is set and 'ref-binding' is empty, the branch name will be used implicitly.
```

Comment:
unclear what `the branch name` refers to; will default to the branch name.

Though makes me wonder if ref-binding is set should the branch name be added by default if not already in the list? Are there cases where we do not want to add it?

Reply:
tbh I prefer to have a possibility to override implicit things.